File: old/Tram/1.py
x = int(raw_input())
cap = 0
maxcap =0
for i in range(x):
	passen = map(int,raw_input().split())
	cap-=passen[0]
	cap+=passen[1]
	if cap>maxcap:
		maxcap = cap
print maxcap

File: old/Tram/1.rb
x = gets.to_i()
cap = 0
maxcap =0
while x>0
	passen = gets.chomp.split()
	cap-=passen[0].to_i
	cap+=passen[1].to_i
	if cap>maxcap
		maxcap=cap
	end
	x-=1
end
puts maxcap